modules/openy_features/openy_media/src/Plugin/Field/FieldFormatter/VideoLinkFormatter.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Drupal\openy_media\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FormatterBase;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\video_embed_field\ProviderManagerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Url;
+
+/**
+ * Plugin implementation of the video field formatter.
+ *
+ * @FieldFormatter(
+ *   id = "openy_video_link",
+ *   label = @Translation("Video link"),
+ *   field_types = {
+ *     "video_embed_field"
+ *   }
+ * )
+ */
+class VideoLinkFormatter extends FormatterBase implements ContainerFactoryPluginInterface {
```

Comment:
This class should live in `openy_media_video`, because it's being used only there.